core/domain/exp_services.py

```diff
@@ -271,9 +271,9 @@ def export_to_zip_file(exploration_id, version=None):
         exploration_id, version=version)
     yaml_repr = exploration.to_yaml()
 
-    memfile = python_utils.string_io()
+    tempfile = python_utils.string_io()
```

Comment:
tempfile is a python stdlib. Maybe temp_file?

Reply:
Done